packages/render-fidelity-tools/src/components/analysis-view.ts

```diff
@@ -48,7 +48,7 @@ export class AnalysisView extends LitElement {
     const channel = new MessageChannel();
 
     this.port = channel.port1;
-    this.worker = new Worker('../../dist/image-comparison-worker.js');
+    this.worker = new Worker('../dist/image-comparison-worker.js');
```

Comment:
I think we might need to come up with a more sophisticated answer here. This path currently works for docs but fails for local tests.

Reply:
Can you be more specific on the repro? It's working fine for me locally.

Reply:
When the current docs deploy script runs, it moves these files into a new folder, and I think this change will break that path. See https://github.com/GoogleWebComponents/model-viewer/blob/115edc9745cec58b9e2f354a1d2b9ca7a1df023a/packages/modelviewer.dev/scripts/ci-before-deploy.sh#L41

I think this creates an issue because once the docs are deployed, the entrypoint HTML file is one directory fewer away from the worker script. So, we need to fix the paths. We could do this with `sed` in the `ci-before-deploy.sh` script as a hack for now.

Reply:
I'll run the script and confirm that this breaks things, hold on.

Reply:
Looks like this is actually fine